CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,94 @@
+# Contributor Code of Conduct
+
+Our [company values](https://auth0.com/careers/culture) guide us in our
```

Comment:
this intro is very corporatey. probably cause its auth0's lol. we can probably remove this.